cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterTwillApplication.java

```diff
@@ -70,16 +70,20 @@
   private static final String NAME = Constants.Service.MASTER_SERVICES;
   private static final String CCONF_NAME = "cConf.xml";
   private static final String HCONF_NAME = "hConf.xml";
-  private static final Set<String> ALL_SERVICES = ImmutableSet.of(
-    Constants.Service.MESSAGING_SERVICE,
-    Constants.Service.TRANSACTION,
-    Constants.Service.DATASET_EXECUTOR,
-    Constants.Service.STREAMS,
-    Constants.Service.LOGSAVER,
-    Constants.Service.METRICS_PROCESSOR,
-    Constants.Service.METRICS,
-    Constants.Service.EXPLORE_HTTP_USER_SERVICE
-  );
+
+  // A map from service name to cConf configuration key prefix for service specific container configuration
+  // They are not necessarily the same (e.g. "transaction" vs "data.tx").
+  // We follow the same naming prefix for configuring containers' memory/vcores.
+  private static final Map<String, String> ALL_SERVICES = ImmutableMap.<String, String>builder()
+    .put(Constants.Service.MESSAGING_SERVICE, "messaging.")
```

Comment:
it seems cleaner to append the `.` in the usage once, instead of appending each service name w/ a dot here.